scrapers/covid-19/getVaccine.js

```diff
@@ -21,6 +25,181 @@ const cleanData = (data) => {
 	}));
 };
 
+const groupVaccineDataByCountry = (vaccineData) => {
+	const groupedByCountryVaccineDataObject = {},
+		worldVaccineDataObject = {},
+		otherJurisdictionsVaccineDataObject = {};
+	vaccineData.forEach((timelineData) => {
+		switch (timelineData.iso_code) {
+			// For jurisidctions like Wales, Scotland, Northern Ireland and England which don't have iso3 code
+			case '':
+				if (otherJurisdictionsVaccineDataObject[timelineData.iso_code] === undefined) {
+					otherJurisdictionsVaccineDataObject[timelineData.iso_code] = [timelineData];
+					return;
+				}
+				otherJurisdictionsVaccineDataObject[timelineData.iso_code].push(timelineData);
+				return;
+				// For world aggregate. World iso3 code is OWID_WRL in data source
+			case 'OWID_WRL':
+				if (worldVaccineDataObject[timelineData.iso_code] === undefined) {
+					worldVaccineDataObject[timelineData.iso_code] = [timelineData];
+					return;
+				}
+				worldVaccineDataObject[timelineData.iso_code].push(timelineData);
+				return;
+				// Countries with iso3 code
+			default:
+				if (groupedByCountryVaccineDataObject[timelineData.iso_code] === undefined) {
+					groupedByCountryVaccineDataObject[timelineData.iso_code] = [timelineData];
+					return;
+				}
+				groupedByCountryVaccineDataObject[timelineData.iso_code].push(timelineData);
+		}
+	});
```

Comment:
This can be simplified: 
```js
const getTimelineDataFromIsoCode = (isoCode) => {
    return switch (isoCode) {
        case '': return otherJurisdictionsVaccineDataObject;
        case: 'OWID_WRL': return worldVaccineDataObject;
        default: return groupedByCountryVaccineDataObject;
    }
}

vaccineData.forEach((timelineData) => {
    const dataObject = getTimelineDataFromIsoCode(timelineData.iso_code);
    if (dataObject[timelineData.iso_code] === undefined) dataObject[timelineData.iso_code] = [timelineData];
    else dataObject[timelineData.iso_code].push(timelineData);
});